sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -0,0 +1,250 @@
+        return Arrays.asList(
+            new TextDocumentInput("0", TestUtils.LINKED_ENTITY_INPUTS.get(0)),
+            new TextDocumentInput("1", TestUtils.LINKED_ENTITY_INPUTS.get(1))
+        );
+    }
+
+    static List<TextDocumentInput> getTextDocumentKeyPhraseInputs() {
+        return Arrays.asList(
+            new TextDocumentInput("0", KEY_PHRASE_INPUTS.get(0)),
+            new TextDocumentInput("1", KEY_PHRASE_INPUTS.get(1))
+        );
+    }
+
+    static List<TextDocumentInput> getTextDocumentSentimentInputs() {
+        return Arrays.asList(
+            new TextDocumentInput("0", TestUtils.SENTIMENT_INPUTS.get(0)),
+            new TextDocumentInput("1", TestUtils.SENTIMENT_INPUTS.get(1))
+        );
+    }
+
+    /**
+     * Helper method to get the expected Batch Detected Languages
+     */
+    static DocumentResultCollection<DetectLanguageResult> getExpectedBatchDetectedLanguages() {
+        DetectedLanguage detectedLanguage1 = new DetectedLanguage("English", "en", 1.0);
+        DetectedLanguage detectedLanguage2 = new DetectedLanguage("Spanish", "es", 1.0);
+        DetectedLanguage detectedLanguage3 = new DetectedLanguage("(Unknown)", "(Unknown)", 0.0);
+        List<DetectedLanguage> detectedLanguageList1 = Collections.singletonList(detectedLanguage1);
+        List<DetectedLanguage> detectedLanguageList2 = Collections.singletonList(detectedLanguage2);
+        List<DetectedLanguage> detectedLanguageList3 = Collections.singletonList(detectedLanguage3);
+
+        TextDocumentStatistics textDocumentStatistics1 = new TextDocumentStatistics(26, 1);
+        TextDocumentStatistics textDocumentStatistics2 = new TextDocumentStatistics(39, 1);
+        TextDocumentStatistics textDocumentStatistics3 = new TextDocumentStatistics(6, 1);
+
+        DetectLanguageResult detectLanguageResult1 = new DetectLanguageResult("0", textDocumentStatistics1, null, detectedLanguage1, detectedLanguageList1);
+        DetectLanguageResult detectLanguageResult2 = new DetectLanguageResult("1", textDocumentStatistics2, null, detectedLanguage2, detectedLanguageList2);
+        DetectLanguageResult detectLanguageResult3 = new DetectLanguageResult("2", textDocumentStatistics3, null, detectedLanguage3, detectedLanguageList3);
+
+        TextDocumentBatchStatistics textDocumentBatchStatistics = new TextDocumentBatchStatistics(3, 3, 0, 3);
+        List<DetectLanguageResult> detectLanguageResultList = Arrays.asList(detectLanguageResult1, detectLanguageResult2, detectLanguageResult3);
+
+        return new DocumentResultCollection<>(detectLanguageResultList, DEFAULT_MODEL_VERSION, textDocumentBatchStatistics);
+    }
+
+    /**
+     * Helper method to get the expected Batch Named Entities
+     */
+    static DocumentResultCollection<RecognizeEntitiesResult> getExpectedBatchNamedEntities() {
+        NamedEntity namedEntity1 = new NamedEntity("Seattle", "Location", null, 26, 7, 0.80624294281005859);
```

Comment:
We should remove all the score equal comparison. Currently. the nightly live test failed because of it.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=242950&view=logs&j=4d5db6ce-0b7f-527e-b115-2367ee6e1fef&t=b7d16dfc-4abf-5ff2-eaa7-e82f3df2ef1b

Reply:
Updated to not check for equality for offset/length/score properties on models.

Reply:
should the score value need to clean up also?

Reply:
same question apply to other hard-coded numerical values